easybuild/easyconfigs/s/Subread/Subread-2.0.0-GCC-10.2.0.eb

```diff
@@ -0,0 +1,44 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'Subread'
+version = '2.0.0'
```

Comment:
@justbennet Is there a reason you're sticking to version 2.0.0 while there's a 2.0.2 release already?

Reply:
Umm, no, not really.  Frankly I did not look.  Mea culpa.

This is the version that we were asked to provide, but frankly, they probably didn't look either.  ;-)

Would you like to reject this PR, and I'll update the version and submit for Subread-2.0.2?